lib/api/v3/notifications/notifications_api.rb

```diff
@@ -47,7 +47,7 @@ def notification_scope
                 .recipient(current_user)
                 .includes(NotificationRepresenter.to_eager_load)
                 .where
-                .not(reason_ian: nil)
+                .not(reason: nil)
```

Comment:
I might be mistaken but I think this should never be the case any more. A notification being created without a reason is invalid, as ensured by the contract.